src/mock/quic_p2p/mod.rs

```diff
@@ -132,11 +134,18 @@ impl QuicP2p {
 }
 
 /// Configuration for `QuicP2p`.
-#[derive(Default, Clone)]
+#[derive(Debug, Default, Clone, Eq, PartialEq, Serialize, Deserialize, StructOpt)]
 pub struct Config {
     /// Hard-coded contacts.
+    #[structopt(
```

Comment:
is that because the real quic_p2p has this?

Reply:
just because the vault code heavily use it.  :(

Reply:
Yep I ran into the same conundrum. I wonder if it's possible to wrap these structs to avoid structopt in routing? I suspect structop might cause a bit of build time overhead?